### DISCUSSION TITLE
The version of streamlit that leafmap depends on is too old?

### DISCUSSION THREAD

**Discussion by swdmike** [opengeos/leafmap#418] [Q&A] [answered]
I don't know if it is my problem or not, but when I try to install leafmap using 'mamba install -c conda-forge leafmap', the downgrade of streamlit seems too much. I have streamlit and streamlit-folium installed.

Downgrade:
----------------------------------------------------------------------------

- folium                         0.14.0  
+ folium                         0.13.0  
conda-forge/noarch                                            68kB
- ipywidgets                      8.0.6  
+ ipywidgets                      7.7.5  
conda-forge/noarch                                           104kB
- jupyterlab_widgets              3.0.7  
+ jupyterlab_widgets              1.1.4  
conda-forge/noarch                                           114kB
- streamlit                      1.21.0  
+ streamlit                       1.9.0  
conda-forge/noarch                                             8MB
- streamlit-folium               0.11.1  
+ streamlit-folium               0.11.0  
conda-forge/noarch                                           359kB
- widgetsnbextension              4.0.7  
+ widgetsnbextension              3.6.4

**Comment by giswqs** [comment, marked as answer]
Leafmap pins the ipywidgets and folium version because folium v0.14.0 has some bugs and VS Code does not support ipywidgets >=0.8.0. Hope to remove these restrictions in the next release. You can use pip install as a workaround if needed.
